interactions/api/models/guild.py

```diff
@@ -2019,6 +2020,157 @@ async def delete_emoji(
             if int(item.id) == int(emoji_id):
                 return self.emojis.remove(item)
 
+    async def get_stickers(self) -> List[Sticker]:
+        """
+        Get the stickers for a guild.
+
+        :return: List of stickers of the guild.
+        :rtype: List[Sticker]
+        """
+        if not self._client:
+            raise LibraryException(code=13)
+
+        res = await self._client.list_guild_stickers(guild_id=int(self.id))
+
+        stickers = [Sticker(**sticker) for sticker in res]
+        self.stickers = stickers
+
+        return self.stickers
+
+    async def get_nitro_sticker_packs(self) -> List[StickerPack]:
+        """
+        Gets the list of sticker packs available to Nitro subscribers.
+
+        :return: List of sticker packs.
+        :rtype: List[StickerPack]
+        """
+        if not self._client:
+            raise LibraryException(code=13)
+
+        res = await self._client.list_nitro_sticker_packs()
```

Comment:
How does this work without guild id?

Reply:
> How does this work without guild id?

Because endpoint doesn't requires guild id
![image](https://user-images.githubusercontent.com/25762802/187593438-35308974-bf9d-4b5b-b5bf-37ad76d07575.png)

Reply:
Oh wait does it just return those standard discord wumpus (and other) stickers?

Reply:
Oh wait does it just return those standard discord wumpus (and other) stickers?

Reply:
yeah